docs/src/main/asciidoc/sqs.adoc

```diff
@@ -611,6 +611,19 @@ Any number of `@SqsListener` annotations can be used in a bean class, and each a
 
 NOTE: Queues declared in the same annotation will share the container, though each will have separate throughput and acknowledgement controls.
 
+===== Annotation-driven SqsListener
```

Comment:
@msosa, I'm not sure I understand this section. I was thinking we could have a `SNS Messages` or similar section describing this feature.

Reply:
Thanks, that makes more sense